core/storage/suggestion/gae_models_test.py

```diff
@@ -6020,3 +6020,15 @@ def test_get_model_association_to_user(self) -> None:
         self.assertEqual(
             model.get_model_association_to_user(),
             base_models.MODEL_ASSOCIATION_TO_USER.ONE_INSTANCE_PER_USER)
+
+
+class SortChoicesUnitTest(test_utils.GenericTestBase):
+    """Test for checking enum SortChoices matches with values in
+    CD_ADMIN_STATS_SORT_OPTIONS in constants.ts.
+    """
+
+    def test_enum_dictionary_match(self) -> None:
+        for enum_member in suggestion_models.SortChoices:
+            self.assertTrue(
+                enum_member.value in constants.CD_ADMIN_STATS_SORT_OPTIONS
+            )
```

Comment:
Check the other way round, too -- anything in that constants file must be in the enum.

I suggest getting the keys of each, sorting them, and making sure those lists are equal.